src/recipes/pompous-mexican

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Pompous Mexican",
+    "description": "A cocktail made from Mexican beer, gin and lemon juice.",
+    "github": "cencristian",
+    "ingredients": [
+        {
+            "quantity": "",
+            "measure": "12 oz",
+            "ingredient": "Corona lager"
+        },
+        {
+            "quantity": "",
+            "measure": "3/4 oz",
+            "ingredient": "Tanqueray gin"
+        },
+        {
+            "quantity": "",
+            "measure": "3/4 oz",
+            "ingredient": "Lemon Juice"
+        }
+    ],
+    "directions": [
+        "1. Take a sip (or pour out) a small amount of the Corona, down to where the neck widens, creating room for the other ingredients."
+        "2. Carefully pour the Tanqueray & lemon juice into the bottle of Corona."
+        "3. Cap the bottle tightly with your thumb, invert until the ingredients mix fully."
+        "4. Drink and enjoy."
```

Comment:
You can remove the numerations here. This array will get rendered as an ordered list.